lib/outbox/bandwidth/client.rb

```diff
@@ -38,7 +38,7 @@ def deliver(sms)
         messaging_client = @api_client.messaging_client.client
         body = ::Bandwidth::MessageRequest.new
         body.application_id = sms[:application_id]
```

Comment:
Should this also be an option you can set during initialization

Reply:
yeah, that would be better. I tried setting it as a default like the from field, but it initializing the client with it and storing it as an instance variable seems better. then we don't have to assign it in perform delivery in notifications (which I am trying to avoid)